varada_trino_manager/infra/call_home_methods.py

```diff
@@ -0,0 +1,242 @@
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+from re import findall as re_findall
+import re
+import os
+import boto3
+import fnmatch
+import gzip
+import io
+import urlpath
+import json
+import collections
+import datetime
+import time
+
+
+client = boto3.client("s3")
```

Comment:
please don't init anything in the "public" namespace.
since client is only used in the S3URL class init the client in the class init
if you think it will create too many copies of client consider using singleton